go.mod

```diff
@@ -55,7 +55,7 @@ require (
 	github.com/stretchr/testify v1.7.1
 	github.com/ugorji/go/codec v1.2.7
```

Comment:
`go.mod` still uses `github.com/onsi/ginkgo` of version ` v1.16.5`. Or do I miss something?

Reply:
UGH I FORGOT ABOUT GO'S SILLY /V2 THING

<fixes>